### PR TITLE
Updated docs

### PR DESCRIPTION
**Pull request by Dmdv**
## Tests 
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: code/go/0chain.net/smartcontract/zcnsc/README.MD [hunk 1]
```diff
@@ -73,28 +73,106 @@ https://mumbai.polygonscan.com/address/0xd37F574927f88566Fe9AE8623Fbb5F85a96b2Dd
...
+
+```text
+Flags:
+--bridge_config string       Bridge config file name (default "bridge.yaml")
+--bridgeaddress string       Bridge contract address
+--consensusthreshold float   Consensus threshold required to reach consensus for burn tickets (default 0.75)
+--ethereumaddress string     Client Ethereum address
+--ethereumnodeurl string     Ethereum Node URL (Infura/Alchemy)
+--gaslimit int               appr. Gas limit to execute Ethereum transaction (default 300000)
+--password string            Password be used to unlock private key stored in local storage
```
Comment (Sriep) [inline code/go/0chain.net/smartcontract/zcnsc/README.MD]: `Password to be used`

File: code/go/0chain.net/smartcontract/zcnsc/README.MD [hunk 2]
```diff
@@ -73,28 +73,106 @@ https://mumbai.polygonscan.com/address/0xd37F574927f88566Fe9AE8623Fbb5F85a96b2Dd
...
+--path string                Configuration dir (default "/Users/dima/.zcn")
```
Comment (Sriep) [inline code/go/0chain.net/smartcontract/zcnsc/README.MD]: dir -> directory

**Review by Sriep — APPROVED** [review]

File: code/go/0chain.net/smartcontract/zcnsc/README.MD (hunk 2 above)
Comment (Dmdv) [reply code/go/0chain.net/smartcontract/zcnsc/README.MD]: fixed

File: code/go/0chain.net/smartcontract/zcnsc/README.MD (hunk 1 above)
Comment (Dmdv) [reply code/go/0chain.net/smartcontract/zcnsc/README.MD]: fixed

**Pull request merged**